framework/wazuh/rule.py

```diff
@@ -156,7 +156,7 @@ def get_rules_files(status=None, path=None, file=None, offset=0, limit=common.da
         # Rules configuration
         ruleset_conf = configuration.get_ossec_conf(section='ruleset')
         if not ruleset_conf:
-            raise WazuhException(1200)
+            raise WazuhInternalError(1200)
```

Comment:
I doubt if this one should be a WazuhError. I don't see the point showing this in the log.